c2cgeoform/ext/deform_ext.py

```diff
@@ -471,13 +471,37 @@ class Photo(models.FileData, Base):
                 'widget': deform_ext.FileUploadWidget(file_upload_temp_store)
             }
             permission_id = Column(Integer, ForeignKey('excavations.id'))
+
+    **Attributes/Arguments**
+
+    get_url (optional)
+        A callback function `function(request, id) -> string` which returns
+        the URL to get the file. Example usage:
+
+        .. code-block:: python
+
+            'widget': deform_ext.FileUploadWidget(
+                _file_upload_temp_store,
+                get_url=lambda request, id: request.route_url('file', id=id)
```

Comment:
Yes it is.
tal:omit-tag only remove the container tag (\<a\>), not the content.